src/Azure.Functions.Cli/Actions/LocalActions/InstallExtensionAction.cs

```diff
@@ -85,39 +85,42 @@ public async override Task RunAsync()
                 return;
             }
 
-            if (CommandChecker.CommandExists("dotnet"))
```

Comment:
I think we still need another if statement that checks for dotnet command and throws an exception with appropriate error message.

Reply:
I moved the check to AddPackage command before we actually run the dotnet command. This way we don't fail if the users don't have dotnet, and we don't end up having to run dotnet commands.

I agree with your comment there that we shouldn't check before every package add. Is there a way to add all the packages with one dotnet command? Running dotnet commands several times probably adds the overhead too.